media-video/obs-studio/obs-studio-9999.ebuild

```diff
@@ -50,15 +50,7 @@ COMMON_DEPEND="
 	v4l? ( media-libs/libv4l )
 "
 DEPEND="${COMMON_DEPEND}
-	imagemagick? (
-		|| (
-			<media-gfx/imagemagick-7
-			(
-				>=media-gfx/imagemagick-7
-				>=dev-util/cmake-3.9
-			)
-		)
-	)
+	imagemagick? ( >=dev-util/cmake-3.9 )
```

Comment:
In addition to dev-util/cmake, USE=imagemagick should pull in:

```
>=media-gfx/imagemagick-7
```

(*dropping pre-v7 support, the ebuild will still need to depend on
imagemagick if building with the imagemagick feature enabled*)